.github/workflows/release.yml

```
@@ -33,6 +33,7 @@ jobs:
         DO_RELEASE=false
         if grep -q '^[0-9]\+-SNAPSHOT$' version; then
           sed -i 's/^\([0-9]\+\)-SNAPSHOT$/\1/' version
+          VERSION=$(sed -n 's/^\([0-9]\+\).*$/\1/p' version | head -n 1)
```

how is it different from

```suggestion
          VERSION=$(cat version)
```

?

---

No different now that the version lives in a file. Thanks, much simpler.